### PR TITLE
Richards mechanics; Bishop's effective stress models

### PR DESCRIPTION
**Pull request by endJunction**
Adding two different models for Bishop's effective stress, the power law and the saturation cut-off.

1. [ ] Feature description was added to the [changelog](https://github.com/ufz/ogs/wiki/Release-notes-6.3.0)
2. [x] Tests covering your feature were added?
3. [x] Any new feature or behavior change was documented?

### CODE REVIEW

File: MaterialLib/MPL/Properties/BishopsPowerLaw.cpp [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+{
+BishopsPowerLaw::BishopsPowerLaw(double const exponent) : _m(exponent) {}
+
+void BishopsPowerLaw::setScale(
+    std::variant<Medium*, Phase*, Component*> scale_pointer)
+{
+    if (!std::holds_alternative<Medium*>(scale_pointer))
+    {
+        OGS_FATAL(
+            "The property 'BishopsPowerLaw' is implemented on the 'media' "
```
Comment (TomFischer) [inline MaterialLib/MPL/Properties/BishopsPowerLaw.cpp]: media or medium scale?

File: MaterialLib/MPL/Properties/BishopsSaturationCutoff.cpp [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+}
+
+void BishopsSaturationCutoff::setScale(
+    std::variant<Medium*, Phase*, Component*> scale_pointer)
+{
+    if (!std::holds_alternative<Medium*>(scale_pointer))
+    {
+        OGS_FATAL(
+            "The property 'BishopsSaturationCutoff' is implemented on the "
+            "'media' scale only.");
```
Comment (TomFischer) [inline MaterialLib/MPL/Properties/BishopsSaturationCutoff.cpp]: Same as above.

File: MaterialLib/MPL/Properties/BishopsPowerLaw.cpp (hunk 1 above)
Comment (endJunction) [reply MaterialLib/MPL/Properties/BishopsPowerLaw.cpp]: So far, only 'media' is used (due to copy-paste)...

**Comment by endJunction** [comment]
@bilke Do you have an idea why the latex code doesn't render correctly [here](https://deploy-preview-2823--ogs.netlify.com/docs/benchmarks/richards-mechanics/bishops-effective-stress/)?

**Pull request merged**